app/src/main/java/com/xmartlabs/template/helper/GeneralErrorHelper.java

```diff
@@ -34,37 +36,71 @@
   @Inject
   SessionController sessionController;
 
+  @Getter
+  private static final RxJavaErrorHandler rxErrorHandler = new RxJavaErrorHandler() {
+    @Override
+    public void handleError(Throwable error) {
+      super.handleError(error);
+      generalErrorAction.call(error);
+    }
+  };
+
   public GeneralErrorHelper() {
     BaseProjectApplication.getContext().inject(this);
   }
 
+  private static void logCrashlyticsError(Response<?> response) {
+    String url = response.raw().request().url().toString();
+    int resultCode = response.code();
+    String headers = response.headers().toString();
+    String body = null;
+    Crashlytics.setString(CRASHLYTICS_KEY_URL, url);
+    Crashlytics.setInt(CRASHLYTICS_KEY_STATUS_CODE, resultCode);
+    Crashlytics.setString(CRASHLYTICS_KEY_RESPONSE_HEADERS, headers);
+    try {
+      body = response.errorBody().string();
+    } catch (IOException e) {
+      Timber.w(e, "Couldn't read error body");
+    }
+    Crashlytics.setString(CRASHLYTICS_KEY_RESPONSE_BODY, body);
```

Comment:
I am not logging `null`, I'm clearing the body key! However, in my opinion it's the same because it will be `null`, so I'll change it now! No problem :+1: